.github/workflows/r_solaris.yml

```diff
@@ -0,0 +1,76 @@
+name: Solaris CRAN check
+
+on:
+  repository_dispatch:
+    types: [gha_run_r_solaris]
+
+jobs:
+  test:
+    name: solaris-cran
+    timeout-minutes: 60
```

Comment:
I run 20+ different checks on various platforms. All of them either finished within around 20 minutes or were aborted on RHub side (whole build took 44 minutes in that case):

```
3527#> * checking for unstated dependencies in ‘tests’ ... OK
3528#> * checking tests ...
3529#> Running ‘testthat.R’Build timed out (after 20 minutes). Marking the build as failed.
3530#> Build was aborted
```

Reply:
I'm surprised, but glad to hear it! I have had experiences before where I submit a job on a less-common platform like Solaris and it runs overnight (maybe 10 hours from submission to result)

Reply:
Maybe there were no timeouts on Rhub before?.. I don't know... You can check "Submitted:" and "Build time:" lines in all e-mails in http://www.yopmail.com/lightgbm_test_email to check response time.

![image](https://user-images.githubusercontent.com/25141164/107132429-23620c00-68f0-11eb-8fcf-52c7c6fb3927.png)